### DISCUSSION TITLE
Border opacity modifier incompatible with css variables

### DISCUSSION THREAD

**Discussion by Timer00** [tailwindlabs/tailwindcss#11759] [Help] [answered]
When using the **opacity modifier** for a custom border-color, the color is simply set to default and the opacity is not reflected.

![image](https://github.com/tailwindlabs/tailwindcss/assets/19307482/1f7a0251-042a-4296-b4a9-700bcb8685f9)

**To reproduce the issue check here:**

https://play.tailwindcss.com/Q1t1lj73Ro

Documentation referenced:

https://tailwindcss.com/docs/border-color#changing-the-opacity

Please let me know if am I doing something dumb or if this is an actual issue.

**Comment by wongjn** [comment, marked as answer]
You'd need to look at reworking your [color CSS variables to work with opacity modifiers](https://tailwindcss.com/docs/customizing-colors#using-css-variables):

```css
:root {
  --color-primary-rgb: 236 72 154;
}
```
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  theme: {
    extend: {
      colors: {
        'primary-fixed': 'rgb(var(--color-primary-rgb) / <alpha-value>)'
      },
    },
  },
}
```
```html
<div class="border-4 border-[rgb(var(--color-primary-rgb)/<alpha-value>)]/10">…</div>
<div class="border-4 border-primary-fixed/10">…</div>
```
[Tailwind Play example](https://play.tailwindcss.com/GPHsaBymmS)
